src/org/thoughtcrime/securesms/loki/activities/SettingsActivity.kt

```diff
@@ -74,14 +74,17 @@ class SettingsActivity : PassphraseRequiredActionBarActivity() {
 
         setContentView(R.layout.activity_settings)
 
+        val origUserDisplayName = DatabaseFactory.getLokiUserDatabase(this).getDisplayName(hexEncodedPublicKey)
```

Comment:
General convention is to follow the indentation style the rest of the file uses

Reply:
Sorry, you mean line breaks or the actual indentation (tabs/spaces)?